src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+import duke.exception.DukeException;
```

Comment:
Not explicitly stated in the Java coding standard but would it be neater and consistent if there is a breakline between package and import?